src/content/docs/en/guides/recipes.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: All Recipes
+description: See guided examples of building features in Astro.
+i18nReady: false
+---
+import RecipeBox from '~/components/recipes/RecipeBox.astro';
+import BackToRecipes from '~/components/recipes/BackToRecipes.astro';
+
+See guided examples of adding features to your Astro project.
+
+
+## Curated Recipes
+
+<RecipeBox icon="clipboard-check">
+  <a href="/en/guides/rss/">Add an RSS feed</a>
+</RecipeBox>
+
+<RecipeBox icon="puzzle-piece">
```

Comment:
This is really three recipes. I wouldn't be afraid of having small recipes that are each their own how-to. We can adjust the UI to have parents/headings.